### ISSUE TITLE
The SQL Server Compact package references an obsolete SQL Server Compact package

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#375]
```
What steps will reproduce the problem?
1. Log into NuGet
2. Locate the package
3. Notice that the package does not reference 
https://www.nuget.org/packages/Microsoft.SqlServer.Compact/ (latest version)

Would be happy to update the nuspec file, but it does not seem tp be part of 
the source code

```

Original issue reported on code.google.com by `ejls...@hotmail.com` on 25 Oct 2014 at 8:14

**Comment by GoogleCodeExporter** [comment]
```
Are you sure you looked into the the right repo[1]?

[1] https://code.google.com/p/elmah/source/browse?repo=pkg
```

Original comment by `azizatif` on 25 Oct 2014 at 10:25

**Comment by GoogleCodeExporter** [comment]
```
Nope, overlooked the repo dropdown. Will update. Should the package version be 
1,2,2,1 or 1,2,3? (I think it must alway be incremented) 
```

Original comment by `ejls...@hotmail.com` on 25 Oct 2014 at 11:03

**Comment by GoogleCodeExporter** [comment]
```
Cannot figure out hg - this is the required change:

      <dependency id="Microsoft.SqlServer.Compact" version="4.0.8876.1" />

and update package version to 1.2.2.1 and publish it
```

Original comment by `ejls...@hotmail.com` on 26 Oct 2014 at 8:02

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e4883b898a07.
```

Original comment by `ejls...@hotmail.com` on 27 Oct 2014 at 8:07
- Changed state: **Fixed**
